pages/challenge.js

```diff
@@ -12,6 +12,32 @@ const RepoLink = 'https://github.com/OperationCode/front-end/';
 const ChallengeLink = `${RepoLink}blob/main/pages/challenge.js`;
 const CompareLink = `${RepoLink}compare`;
 
+export const MakeColumnsFromArray = ({ ary }) => {
```

Comment:
Consider a rename to `NameColumns` because `MakeColumnsFromArray` sounds like a reusable function. 

- It's not reusable (because it has business logic and because it's impure - this sounds bad/negative, but there's no such connotation - it just is what it is).

- While all JSX elements are functions, we typically don't name them like functions because they're supposed to _look_ like structured content. For example, nobody write a CreateButton component (`<CreateButton>`)... well, I guess they could, but only if it's a button that creates something when clicked! I hope you get my point.